loopback.cc

```diff
@@ -55,7 +62,12 @@ int loopback_init(const char* device, int w, int h, int debug) {
 	vid_format.fmt.pix.colorspace = V4L2_COLORSPACE_SRGB;
 
 	ret_code = ioctl(fdwr, VIDIOC_S_FMT, &vid_format);
-	assert(ret_code != -1);
+	if(ret_code < 0) {
+		perror(__func__);
+		printf("The device video format could not be set.\n");
+		close(fdwr);
+		return -1;
+	}
 
```

Comment:
As above, consistent output to `stderr` and line number, or unique descriptive message for identification.